custom/enikshay/integrations/bets/repeater_generators.py

```diff
@@ -322,6 +323,9 @@ def create_voucher_payload(cls, voucher_case):
         approver_name = approver.name
         usertype = approver.user_data.get('usertype')
         approver_usertype = USERTYPE_DISPLAYS.get(usertype, usertype)
+        amount = voucher_case_properties.get(AMOUNT_APPROVED, "")
+        if amount == "":
```

Comment:
nit: `if not amount` would do the same.

Reply:
that wouldn't work if `amount == 0` or `0.0`